README.md

```diff
@@ -100,32 +106,37 @@ rss.device.monitor.enabled false
 
 EXAMPLE: HA cluster
 ```properties
+# used by client and worker to connect to master
+celeborn.master.endpoints=clb-1:9097,clb-2:9098,clb-3:9099
+
+# used by master nodes to bootstrap, every node should know the topology of whole cluster, for each node,
+# `celeborn.ha.master.node.id` should be unique, and `celeborn.ha.master.node.<id>.host` is required
+celeborn.ha.enabled true
+celeborn.ha.master.node.id 1
+celeborn.ha.master.node.1.host clb-1
+celeborn.ha.master.node.1.port 9097
+celeborn.ha.master.node.1.ratis.port 9872
+celeborn.ha.master.node.2.host clb-2
+celeborn.ha.master.node.2.port 9098
+celeborn.ha.master.node.2.ratis.port 9873
+celeborn.ha.master.node.3.host clb-3
+celeborn.ha.master.node.3.port 9099
+celeborn.ha.master.node.3.ratis.port 9874
```

Comment:
The ideas here are

1. service id (previously defined by `rss.ha.service.id`) is removed

2. introduces pattern `celeborn.ha.master.node.<id>.<key>` for each master node, and the full node id list can be extracted from the configuration keys, so the previous `rss.ha.nodes.dev-cluster` is not required anymore. this mechanism requires at least one property to appear for each node, here the `host` is required.

3. previously, the node id is inferred by hostname testing, it simplifies the configuration for the cluster as we don't need to supply node-specific values. but, the backward is we can NOT launch 2 instances on a single node, cause hard to write a UT/IT then. the new approach reserves the infer mechanism, and supplies a new optional configuration `celeborn.ha.master.node.id` to set the node id explicitly. 

4. previously, all nodes of the cluster should use the same port, the new approach removes the restrictions.